literature.bib

```diff
@@ -372,7 +367,29 @@ @mastersthesis{Guething23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2024 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{FK:POS24,
+	title       = {{Life Span of SAT Techniques}},
+	author      = {Mathias Fleury and Daniela Kaufmann},
+	booktitle   = POS,
+	institution = arXiv,
+	doi         = {10.48550/arXiv.2402.01202},
+	year        = 2024,
+	ek-tags     = {satisfiability, preprocessing}
+}
```

Comment:
Other entries for arXiv papers use either @techreport or @article. Would be good to have some consistency here :)
See KTS+:TR22 or SNP:CoRR18 for examples.

Reply:
it is a preprint that will be available in the proceedings, so I'm not sure how to file this

Reply:
then @techreport seems to be correct here
if there is a newer version later on, we can use the `subsumedby` field

Reply:
on that note, the second entry you mentioned (SNP:CoRR18) seems incorrect, as it has been properly published in an Elsevier journal: https://www.sciencedirect.com/science/article/pii/S147784241630015X shouldn't we use that instead of the arXiv version?

Reply:
yes, in that case we should :)
It maybe applies to other arXiv papers too. This was just the first I found that uses @article

Reply:
> then @TechReport seems to be correct here if there is a newer version later on, we can use the `subsumedby` field

I was wrong, this was from last year's workshop. I updated the entry

Reply:
> yes, in that case we should :) It maybe applies to other arXiv papers too. This was just the first I found that uses @Article

should I update the key as well? this could break things

Reply:
is this the same publication?
In this case yes, you can rename the key and use the field `renamedFrom = {SNP:CoRR18}` to indicate that.
If no, then create a second entry and add the fields `subsumes` and `subsumedBy` in the respective entries.